src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -248,6 +248,12 @@ internal bool IsIntegrationEnabled(string name)
             return TraceEnabled && !disabled;
         }
 
+        internal bool IsOptInIntegrationEnabled(string name)
```

Comment:
What do you think of reusing the integration infrastructure and the name `IsOptInIntegrationEnabled` for checking that an integration is explicitly enabled?